mlrun/datastore/v3io.py

```diff
@@ -91,8 +95,33 @@ def get_storage_options(self):
             v3io_api=mlrun.mlconf.v3io_api,
         )
 
+    def upload_helper(self, key, src_path, max_chunk_length=one_GB):
+        file_size = os.path.getsize(src_path)
+        if file_size <= one_MB:
+            http_upload(self.url + self._join(key), src_path, self.headers, None)
+            return
+        append_header = deepcopy(self.headers)
+        append_header["Range"] = "-1"
+        with open(src_path, "rb") as file_obj:
+            file_offset = 0
+            while file_offset < file_size:
+                chunk_len = min(file_size - file_offset, max_chunk_length)
```

Comment:
seems like there is a mix between `size` and `length` when referring to file size, let's try to be more consistent or if you think each of them has different meaning add a comment explaining that

Reply:
done